build-logic/src/main/kotlin/polaris-java.gradle.kts

```diff
@@ -82,7 +82,7 @@ testing {
       dependencies {
         implementation(project())
         implementation(testFixtures(project()))
-        runtimeOnly(
+        implementation(
```

Comment:
why is this change required? Why not add it as an explicit "test" dependency only where it is truly needed?

Reply:
It was previously runtimeOnly so I wasn't sure if some other non-test thing needed it, and I'm not sure if there's a better common ancestor of runtimeOnly/testImplementation.

Reply:
@snazy WDYT about this change?